### PR TITLE
Added new ruleset - Cheapshark.com.xml

### PR DESCRIPTION
**Pull request by RemakingEden**
Closes https://github.com/EFForg/https-everywhere/issues/17214

### CODE REVIEW

**Comment by pipboy96** [comment]
@RemakingEden I recommend you to work on #17436. It's impossible for one person to go through every mixedcontent ruleset.

**Comment by RemakingEden** [comment]
@pipboy96 Thanks, is that related to this pull or just a general request? I'm a little confused. Should I have known to work on that? You make it sound like I should.

**Comment by pipboy96** [comment]
No, I'm not a member of EFF, and I'm not speaking officially.

File: src/chrome/content/rules/Cheapshark.com.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!--
+	Nonfunctional hosts in *.cheapshark.com:
+
+	h: http redirect
```
Comment (J0WI) [inline src/chrome/content/rules/Cheapshark.com.xml]: They don't redirect to `http`on the same domain, so they can be added as target.

Comment (RemakingEden) [reply src/chrome/content/rules/Cheapshark.com.xml]: Okay my bad, I did this before I got your message about http redirects. I think I understand now. Thanks J0WI

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**